aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -994,6 +1000,7 @@ impl AptosVM {
                 .map_err(|err| Self::metadata_validation_error(&err.to_string()))?;
         }
         verifier::resource_groups::validate_resource_groups(session, modules)?;
+        verifier::event_validation::validate_module_events(session, modules)?;
```

Comment:
As we just discussed offline, this needs to be feature gated

Reply:
As just discussed offline, line 1000 is already feature gated so this line is good.